src/api/dataFetching.js

```diff
@@ -5,76 +5,72 @@ const request = async (url) => {
 };
 
 const save = async (item) => {
-  return axios.put(process.env.VUE_APP_PUT_REQUEST + item["o:id"], item, {
+  return axios.put(process.env.VUE_APP_DIRECT_REQUEST + item["o:id"], item, {
     params: {
       key_identity: process.env.VUE_APP_KEY_IDENTITY,
       key_credential: process.env.VUE_APP_KEY_CREDENTIAL,
     },
   });
 };
 
+const parseId = (element) => {
+  return element["o:id"];
+};
+
 const parseElement = (element) => {
   return element["thumbnail_display_urls"];
 };
 
 const parseImages = (data) => {
   const images = [];
   data.forEach((element) => {
-    element["dcterms:rights"] = [
-      {
-        type: "literal",
-        property_id: 15,
-        property_label: "Rights",
-        is_public: true,
-        "@value": "Pending",
-      },
-    ];
-    element["dcterms:educationLevel"] = [
-      {
-        type: "literal",
-        property_id: 46,
-        property_label: "Audience Education Level",
-        is_public: true,
-        "@value": Date.now().toString(),
-      },
-    ];
-    save(element);
-    images.push({ image: parseElement(element), element: element });
+    images.push({
+      image: parseElement(element),
+      id: parseId(element),
+      element: element,
+    });
   });
   return images;
 };
 
-const getPendingImages = async () => {
-  const req = process.env.VUE_APP_FIND_DATA;
-  const { data } = await request(req);
-  data.forEach((element) => {
-    if (+element["dcterms:educationLevel"][0]["@value"] + 300000 < Date.now()) {
-      element["dcterms:rights"] = [];
-      element["dcterms:educationLevel"] = [];
-      save(element);
+const countImageAndTags = (data) => {
+  const nbImage = data.length;
+  let nbTags = 0;
+  data.forEach((image) => {
+    if (image["dcterms:rights"][0]["@value"] === "Save") {
+      nbTags = nbTags + image["dcterms:coverage"].length;
     }
   });
+  return { nbImage: nbImage, nbTags: nbTags };
 };
 
 export default {
-  async getImages(listId) {
-    let req = process.env.VUE_APP_FETCH_BASE;
-    let position = 3;
-    listId.forEach((element) => {
+  async getImages(user, skipId) {
+    let req =
+      process.env.VUE_APP_FETCH_BASE +
+      "per_page=1&property%5B0%5D%5Bjoiner%5D=and&property%5B0%5D%5Bproperty%5D=15&property%5B0%5D%5Btype%5D=nex&property%5B1%5D%5Bjoiner%5D=and&property%5B1%5D%5Bproperty%5D=6&property%5B1%5D%5Btype%5D=eq&property%5B1%5D%5Btext%5D=" +
```

Comment:
would be much better to use axios here with object `params` to follow the parameter you sent. It is hard to know here with parameters hidden in the string + html encoding:
Same for others GET